experimental/packages/otlp-proto-exporter-base/package.json

```diff
@@ -68,9 +68,9 @@
     "@opentelemetry/api": "^1.0.0"
```

Comment:
not blocking but I noticed that there is no `peer-api-check` script here. Is this because it's in experimental?

Reply:
No it's probably because the pR that created this package was in parallel with the peer api check PR